.github/workflows/ci.yaml

```diff
@@ -26,10 +26,10 @@ on:
     - cron: '0 3 * * *'  # Every day at 3 AM
   workflow_dispatch:
 
-# Cancels any in-progress runs within the same group identified by workflow name and GH reference (branch or tag) 
+# Cancels any in-progress runs within the same group identified by workflow name and GH reference (branch or tag)
```

Comment:
It would be nice to avoid diffs on unrelated files 😁

Reply:
Will try to fix these, but it happens because of the `.vscode-template` we have in the repo and `"files.trimTrailingWhitespace": true`